processor/k8sattributesprocessor/internal/kube/kube.go

```diff
@@ -241,6 +245,8 @@ type ExtractionRules struct {
 
 	Annotations []FieldExtractionRule
 	Labels      []FieldExtractionRule
+
+	Service bool
```

Comment:
Just `Service` can be misleading. Can we use sth more explicit like `ServiceAttributes` instead?